### ISSUE TITLE
running upgrade.sh on a fresh install fails with 'django.db.utils.ProgrammingError: operator does not exist: character varying >= integer'

### ISSUE THREAD

**Issue by austin987** [netbox-community/netbox#4447]
### Environment
* Python version:  Python 3.6.10
* NetBox version:  2.7.11
* Postgresql version: 10

### Steps to Reproduce
```
#!/bin/bash
set -e
set -x

postgresql="10"
release="2.7.11"

/etc/init.d/postgresql-${postgresql} stop zap

rm -rf /opt/netbox*
cd /opt || exit 1

# get package
wget -c https://github.com/netbox-community/netbox/archive/v${release}.tar.gz
tar xf v${release}.tar.gz
ln -sf netbox-${release} netbox

# setup symlinks for config files
ln -s /etc/netbox/gunicorn_config.py /opt/netbox/gunicorn_config.py
ln -s /etc/netbox/configuration.py /opt/netbox/netbox/netbox/configuration.py

cd /opt/netbox || exit 1

# postgresql (wipe/reinit)
rm -rf /var/lib/postgresql/${postgresql}/data
emerge --config dev-db/postgresql:${postgresql}
/etc/init.d/postgresql-${postgresql} restart
# Setup netbox users/roles
bash -x /usr/bin/initialize-netboxdb

# run upgrade.sh:
bash -x upgrade.sh
```

### Expected Behavior
Netbox configuration finishes and is usable

### Observed Behavior
Traceback from: django
```
Traceback (most recent call last):
  File "netbox/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/core/management/commands/migrate.py", line 234, in handle
    fake_initial=fake_initial,
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/migrations/executor.py", line 245, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/migrations/operations/fields.py", line 249, in database_forwards
    schema_editor.alter_field(from_model, from_field, to_field)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/base/schema.py", line 535, in alter_field
    old_db_params, new_db_params, strict)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/postgresql/schema.py", line 124, in _alter_field
    new_db_params, strict,
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/base/schema.py", line 685, in _alter_field
    params,
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/base/schema.py", line 137, in execute
    cursor.execute(sql, params)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/cacheops/transaction.py", line 99, in execute
    result = self._no_monkey.execute(self, sql, params)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/netbox-2.7.11/venv/lib64/python3.6/site-packages/django_prometheus/db/common.py", line 67, in execute
    return super(CursorWrapper, self).execute(*args, **kwargs)
django.db.utils.ProgrammingError: operator does not exist: character varying >= integer
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
```

Extra info:
This worked with 2.6.8. I bisected the regression, which points to https://github.com/netbox-community/netbox/commit/e09ad6915fd1702b3e06592f3cdf30b601b9f346#

I've reproduced this with 2.7.8 and 2.7.11. In addition, I tried all available stable postgresql versions in gentoo, but behavior was the same, so I don't think that's the issue:
```
dev-db/postgresql-10.9
dev-db/postgresql-11.4
dev-db/postgresql-11.6
dev-db/postgresql-11.7
dev-db/postgresql-12.1
dev-db/postgresql-9.4.22
dev-db/postgresql-9.4.25
dev-db/postgresql-9.5.17
dev-db/postgresql-9.5.18
dev-db/postgresql-9.5.20
dev-db/postgresql-9.6.13
dev-db/postgresql-9.6.14
dev-db/postgresql-9.6.16
dev-db/postgresql-9.6.17
```

Also attach the full output of the reproduction script (with set -x enabled) for reference:
[netbox-2.7.11.txt](https://github.com/netbox-community/netbox/files/4423978/netbox-2.7.11.txt)

**Comment by jeremystretch** [comment]
The upgrade script is provided for upgrading an existing release only. New installations must be performed according to the [installation documentation](https://netbox.readthedocs.io/en/stable/installation/).

**Comment by austin987** [comment]
For anyone else finding this, retesting with 2.8.0 worked for me, git bisect shows this was the fix:
```
01b9d1a4939bcf17cb2c37c6e65d619ffd1388f5 is the first bad commit
commit 01b9d1a4939bcf17cb2c37c6e65d619ffd1388f5
Author: Jeremy Stretch <stretch@packetlife.net>
Date:   Tue Feb 18 16:03:28 2020 -0500

    Closes #3848: Upgrade to Django 3.0

 netbox/dcim/migrations/0097_mptt.py | 28 ++++++++++++++++++++++++++++
 requirements.txt                    |  6 +++---
 2 files changed, 31 insertions(+), 3 deletions(-)
```
